src/main/java/pl/kodujmy/music/music/MusicController.java

```diff
@@ -0,0 +1,14 @@
+package pl.kodujmy.music.music;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MusicController {
+
+    @RequestMapping("/")
```

Comment:
Czy aby na pewno chcielibyśmy wystawiać to pod "/"?

Może lepiej byłoby "/home"